### ISSUE TITLE
Cannot find name 'ArgsType'.

### ISSUE THREAD

**Issue by moranje** [EpicEric/jest-mock-process#7]
The following definition is missing a type definition (`ArgsType`), unless I am missing something.

The typescript compiler is is complaining because I'm using the `spyOnImplementing` function in my test suite. I haven't got a clue what ArgsType is supposed to be though. 

``` ts
export declare function spyOnImplementing<T extends object, M extends FunctionPropertyNames<T>, F extends T[M], I extends (...args: any[]) => ReturnType<F>>(target: T, property: M, impl: I): jest.SpyInstance<ReturnType<F>, ArgsType<F>>;
```

**Comment by EpicEric** [comment]
Oh... seems like it's a type only available in `@types/jest`, and not a standard Typescript type.

I've copied the definition into the library and released patch 1.3.1. Please see if it solves your problem.

**Comment by moranje** [comment]
Thanks, that did it!
